collect_app/src/main/java/org/odk/collect/android/utilities/MyanmarDateUtils.java

```diff
@@ -59,10 +61,17 @@ public static MyanmarDate createMyanmarDate(int myanmarYear, int myanmarMonthInd
     }
 
     public static String[] getMyanmarMonthsArray(int myanmarYear) {
-        return MyanmarDateKernel
-                .getMyanmarMonth(myanmarYear, 1)
-                .getMonthNameList(new LanguageCatalog(Language.MYANMAR))
-                .toArray(new String[0]);
+        if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            return MyanmarDateKernel
+                    .getMyanmarMonth(myanmarYear, 1)
+                    .getMonthNameList(new LanguageCatalog(Language.MYANMAR))
```

Comment:
Since only this part depends on the os version (`Language.MYANMAR` or `Language.ENGLISH`) I would factor out a method that would return `Language` based on os version and call it here.

Reply:
Hie @grzesiek2010, I got your point, writing the same return statement twice just for a change in a single variable seems a bit cumbersome to me as well. How about, if I create a separate bool variable `higherVersion` to check the version, and then use the ternary operator to get the correct language on the system.

Reply:
Ternary operator would bake that line long I would prefer having a separate method it could return Language or even LanguageCatalog and then the line would be:
getMonthNameList(getLanguageCatalog())